<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigTab</class>
 <widget class="QWidget" name="ConfigTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/images/ntfy-nobg.svg</normaloff>:/images/ntfy-nobg.svg</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <layout class="QFormLayout" name="centralWidget">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="placeholderText">
        <string>A name for your server.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="serverLabel">
       <property name="text">
        <string>Server</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="serverLineEdit">
       <property name="placeholderText">
        <string>ntfy.sh (only the domain)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="topicLabel">
       <property name="text">
        <string>Topic</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="topicLineEdit">
       <property name="placeholderText">
        <string>someTopicName</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
